store/tikv/delete_range_test.go

```diff
@@ -0,0 +1,139 @@
+// Copyright 2018 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package tikv
+
+import (
+	"bytes"
+	"context"
+	"math/rand"
+	"sort"
+
+	. "github.com/pingcap/check"
+	"github.com/pingcap/tidb/store/mockstore/mocktikv"
+)
+
+type testDeleteRangeSuite struct {
+	OneByOneSuite
+	cluster *mocktikv.Cluster
+	store   *tikvStore
+}
+
+var _ = Suite(&testDeleteRangeSuite{})
+
+func (s *testDeleteRangeSuite) SetUpTest(c *C) {
```

Comment:
CI fail because OneByOneSuite lock and unlock mismatch.
SetUpTest should be SetUpSuite here. @MyonKeminta